### ISSUE TITLE
Block team chat messagaes.

### ISSUE THREAD

**Issue by Skullians** [fooooooooooooooo/VelocityDiscord#58]
Hey, we're using "team" plugins which have a chat toggle functionality, where all chats sent are only sent to members of the team.
However, these are still sent to the chatbridge discord channel.
Is there any way to fix this?

**Comment by fooooooooooooooo** [comment]
the way messages are collected and sent to discord is by watching for the message sent event emitted by velocity

to filter messages from being sent to discord, velocitydiscord needs a way to know whether to send the message or not

if the plugin you are using has an api to check that or even just has a way to get that information out of the internals of the plugin then its possible

i should come up with a way for other plugins to register themselves with velocitydiscord or something to allow the plugins to do the implementation rather than adding compatibility myself, but i cant think of a good way to do that yet

for now, manually adding compatibility with every plugin that needs it is necessary

**Comment by Skullians** [comment]
I assume the teams plugin would have to be on the proxy then in order to interact with the team plugin's API, if any?

**Comment by fooooooooooooooo** [comment]
yes, unless it or another plugin used a messaging channel to send the data to velocity
